python/ml_croissant/scripts/migrations/migrate.py

```diff
@@ -0,0 +1,20 @@
+"""migrate module."""
+
+import json
+
+from etils import epath
+from ml_croissant._src.core.json_ld import expand_json_ld, reduce_json_ld
+
+if __name__ == "__main__":
+    datasets = [
+        path for path in epath.Path("../../datasets").glob("*titanic*/metadata.json")
```

Comment:
Why just titanic?

Reply:
Acknowledged.

For the moment, I only did one dataset.